src/app/components/additional/additional.component.html

```diff
@@ -0,0 +1,59 @@
+<div class="additional">
+    <evt-ms-desc-section [label]="'additional'"></evt-ms-desc-section>
+    <ng-container *ngIf="data?.adminInfo.structuredData; else elseBlock">
+        <div class="flex-container" *ngIf="data?.adminInfo?.recordHist">
+            <div class="flex-item-label">
+                <evt-ms-desc-section [label]="'recordHist'" [inlineLabel]="true"></evt-ms-desc-section>
+            </div>
+            <div class="flex-item-information">
+                <evt-content-viewer [content]="data.adminInfo.recordHist.source" class="projectInfo-block"></evt-content-viewer> 
+                <evt-content-viewer *ngFor="let c of data.adminInfo.recordHist.change" [content]="c"></evt-content-viewer>
```

Comment:
`<change>` element is an unhandled element. Currently the visualization of unhandled elements is not optimized, because they are all displayed inline and there are no spaces between them.  I think we can change it in the following way (example for this case):
- Remove `flat()` from `unhandledElement()` function in parser-models.ts
- Change `change?: Array<ParseResult<GenericElement>>;` in `changes?: Array<ParseResult<GenericElement>>;`
- In template will be 
```
<div *ngFor="let changes of data.adminInfo.recordHist.change">
      <evt-content-viewer *ngFor="let change of changes" [content]="change"></evt-content-viewer>
</div>
```

In this way the visualization of unhandled elements wil be better and you won't need too many changes when adding the parser.

@szenzaro do you think it can work?
This optimization can be handled in another pr.